README.md

```diff
@@ -149,6 +150,15 @@ resource "forgejo_repository" "mirror" {
 }
 ```
 
+A **branch protection** can be created like so:
+
+```terraform
+resource "forgejo_repository_branch_protection" "main" {
+  branch_name   = "main"
+  repository_id = forgejo_repository.test_repo.name
```

Comment:
The `forgejo_repository.test_repo` resource hasn't been defined in this example. Also, you'll need to reference the `id`, not the `name` attribute...

I'd suggest to remove this example from the README altogether. You have good examples in the `examples/` folder. The README doesn't need to list examples for all possible resources...